.github/workflows/apidiff.yml

```diff
@@ -6,11 +6,15 @@ on:
   pull_request:
 
 jobs:
+  check_docs_only:
+    uses: kubernetes-sigs/kubebuilder/.github/workflows/check_docs_only.yml@master
```

Comment:
we cannot move in this way. 
That would mean that it would not work in other branches. 

@NikhilSharmaWe 
/hold